milestones.go

```diff
@@ -212,3 +212,34 @@ func (s *MilestonesService) GetMilestoneIssues(pid interface{}, milestone int, o
 
 	return i, resp, err
 }
+
+// GetMilestoneMergeRequestsOptions represents the available GetMilestoneMergeRequests() options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/milestones.html#get-all-merge-requests-assigned-to-a-single-milestone
+type GetMilestoneMergeRequestsOptions ListOptions
+
+// GetMilestoneMergeRequests gets all merge requests assigned to a single project milestone.
```

Comment:
Same here about the comment length. I try to keep general comments to < 80 and code < 100, so would be great if you could do the same 👍

Reply:
Fixed